src/components/Comments/ReplyCommentItem.js

```diff
@@ -149,12 +149,12 @@ const ReplyCommentItem = ({
 
   const voteStyle = () => {
     if (totalVote > 0) {
-      return COLORS.anon_primary;
+      return COLORS.upvote;
     }
     if (totalVote < 0) {
-      return COLORS.redalert;
+      return COLORS.downvote;
     }
-    return COLORS.balance_gray;
+    return COLORS.gray400;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining the color based on `totalVote` seems fine. The change in color constants should not affect the functionality as long as these new color constants (`COLORS.upvote`, `COLORS.downvote`, `COLORS.gray400`) are defined and valid.
```diff
-      return COLORS.anon_primary;
+      return COLORS.upvote;

-      return COLORS.redalert;
+      return COLORS.downvote;

-    return COLORS.balance_gray;
+    return COLORS.gray400;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->